ira-task-1/index.js

```diff
@@ -0,0 +1,7 @@
+let button = document.getElementById("btn");
+
+button.addEventListener("click", function () { 
+    document.body.appendChild(document.createElement("p"));
+    let paraTag = document.getElementsByTagName("p")[0];

```

Comment:
It's a better idea to use ids to get elements rather than by tag name when selecting a particular one.

Reply:
I know that. But then I would have to add an attribute, right? Because I am creating it. I don't know how to give it an id when I am creating it in JS